QUEST.tsv

```diff
@@ -1232,8 +1232,8 @@ QUEST_20150317_001232	And so, some people believed Modestas, having undertaken t
 QUEST_20150317_001233	{memo X}Since that time, whenever Modestas took on a wager, he always betted the same thing. The answer to the question of who the Statue of Klaipeda honored.{nl}To uncover the name of the Goddess, numerous people challenged Modestas to a bet.{nl} {nl} But no matter the impossible odds, Modestas never lost a bet. Therefore nobody was able to win the answer from Modestas for years and time flew by.{nl}
 QUEST_20150317_001234	Though Modestas died an undefeated, invincible gambler, by then he had as mentioned never bet anything other than the Goddess' identity, living purely off the profits gained from such bets through the end of his life. {nl}
 QUEST_20150317_001235	{memo X}In the end, this unknown Goddess, in exchange for giving income to a certain Gambler from Klaipeda, created an interesting, mysterious, yet beautiful statue for the city of Klaipeda, and a tale to last for times to come.{nl} {nl}
-QUEST_20150317_001236	Talk To Knight Titas
-QUEST_20150317_001237	To the Scout
+QUEST_20150317_001236	Talk to Knight Titas
+QUEST_20150317_001237	$Talk to the Scouts
 QUEST_20150317_001238	Road to Klaipeda
 QUEST_20150317_001239	To the Battle Commander
```

Comment:
Even though the original says "to" while you've changed the others to "Talk to", I guess it's fine, but in order to keep the consistency, you should change this line as well.